src/cljs/rems/administration/status_flags.cljs

```diff
@@ -50,7 +50,8 @@
 
 (defn- format-update-error [{:keys [type catalogue-items resources workflows]}]
   (let [language @(rf/subscribe [:language])]
-    [:p (text type)
+    [:<>
```

Comment:
Doesn't this produce a "<<>> </<>>" tag pair in html?

Reply:
It produces a React fragment, i.e. no tag.